INSTALL.md

```diff
@@ -249,12 +252,28 @@ deb-src http://llvm.org/apt/$VER/ llvm-toolchain-$VER-3.7 main" | \
 wget -O - http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
 sudo apt-get update
 
-# All versions
+# All versions (Intel)
 sudo apt-get -y install bison build-essential cmake3 flex git libedit-dev \
   libllvm3.7 llvm-3.7-dev libclang-3.7-dev python zlib1g-dev libelf-dev
-
-# For Lua support
+  
+# For Lua support (Intel)
 sudo apt-get -y install luajit luajit-5.1-dev
+
+# Xenial 16.04 LTS on arm64
+#
+# The arm64 system llvm and clang are not new enough to build this, and at the
+# moment LLVM 6.0.0 is not yet released. Until it is, build it from source
+# according to the supplied directions at http://llvm.org/
+#
```

Comment:
You can add the following to the above "or using the instruction at below subsection `Build LLVM and Clang development libs` with `-DLLVM_TARGETS_TO_BUILD="BPF;AArch64"`. My format is probably not
correct. Please adjust properly.